packages/renderer/src/lib/network/NetworkDetailsSummary.spec.ts

```diff
@@ -0,0 +1,86 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import { router } from 'tinro';
+import { expect, test, vi } from 'vitest';
+
+import NetworkDetailsSummary from './NetworkDetailsSummary.svelte';
+import type { NetworkInfoUI } from './NetworkInfoUI';
+
+const network: NetworkInfoUI = {
+  id: '123456789012345',
+  shortId: '123456789012',
+  name: 'Network 1',
+  driver: 'bridge',
+  created: '2025-10-15T10:30:00.000Z',
+  engineId: 'podman1',
+  engineName: 'Podman 1',
+  engineType: 'podman',
+  selected: false,
+  status: 'UNUSED',
+  ipv6_enabled: false,
+  containers: [
+    { id: 'container1', name: 'Container 1' },
+    { id: 'container2', name: 'Container 2' },
+  ],
+};
+
+vi.mock('tinro', () => {
```

Comment:
AFAIK automocking vi.mock('tinro') should be enough but looks like it's a copy/paste from exiting code